unitary/quantum_chess/quantum_moves.py

```diff
@@ -179,25 +179,50 @@ def split_slide_zero_one(squbit, tqubit_clear, tqubit, path_qubit):
     yield (cirq.ISWAP(squbit, tqubit) ** 0.5).controlled_by(path_qubit)
     yield cirq.X(path_qubit)
     # Note that the following steps are equivalent to
-    # ISWAP(squbit, tqubit_clear).controlled_by(path_qubit)
-    # ISWAP(squbit, tqubit_clear).anti_controlled_by(path_qubit)
-    yield cirq.ISWAP(squbit, tqubit_clear) ** 0.5
-    yield cirq.ISWAP(squbit, tqubit_clear) ** 0.5
+    # ISWAP(squbit, tqubit2).controlled_by(path_qubit)
+    # ISWAP(squbit, tqubit2).anti_controlled_by(path_qubit)
+    yield cirq.ISWAP(squbit, tqubit2)
 
 
-def split_slide_zero_multiple(squbit, tqubit_clear, tqubit, path_ancilla):
+# Same as split_slide_one_zero except path qubit is inverted
```

Comment:
I see the docstring as being about how the function should be used, while the above comment is about implementation. WDYT?

Reply:
Do you think it might be better to have a parameter such as `invert_path_ancilla` instead?  I am starting to get confused with the naming schemes here, e.g. merge_slide_one_zero, merge_slide_zero_one, merge_slide_zero_multiple, etc.  Either that, or maybe we should make the function names more clear?

Reply:
I thought about a parameter, but I don't like how it adds another layer of abstraction while not making the code that much shorter.

The names now at least match up to the comments like `(0, 2+)` in the code that calls them. Would `split_slide_t1_zero_blockers_t2_multiple_blockers` / `merge_slide_s1_one_blocker_s2_multiple_blockers` be any clearer?